packages/twenty-front/src/modules/ui/input/editor/components/CustomAddBlockItem.tsx

```diff
@@ -0,0 +1,42 @@
+import { blockSchema } from '@/activities/blocks/schema';
+
+import { useComponentsContext } from '@blocknote/react';
+
+type CustomAddBlockItemProps = {
+  editor: typeof blockSchema.BlockNoteEditor;
+  children: React.ReactNode; // Adding the children prop
+};
+
+type ContentItem = {
+  type: string;
+  text: string;
+  styles: any;
+};
+export const CustomAddBlockItem = ({
+  editor,
+  children,
+}: CustomAddBlockItemProps) => {
+  const Components = useComponentsContext();
```

Comment:
@FelixMalfait 
this should've been `const Components = useComponentsContext()!` to remove ts errors.

Reply:
Ah damn, I don't really like the `const Components = useComponentsContext()!` but should have done an `if() + return` to avoid the scenario where it's null. Would you mind raising a PR for this? Otherwise I will do it today! Thanks a lot!

Reply:
I will raise it dw. Will also remove some of the entityId ts errors. ps: there are 3 of them
Will clean it up

Reply:
Thanks!!!